package/system/fstools/patches/010-glob-onlydir.patch

```diff
@@ -0,0 +1,158 @@
+ hZmwfpQaXXTfmlqZ00UPWV2RxfiQVJQ=
+X-Google-Smtp-Source: ADFU+vvUm0GVy0k7qcQrbKQu+F0aHbikoPckk01Nme2T66IA7wYa2FEgPh0yte0IJ+luliBwK9tokg==
+X-Received: by 2002:a17:902:7c84:: with SMTP id
+ y4mr5677302pll.30.1585178869431; 
+ Wed, 25 Mar 2020 16:27:49 -0700 (PDT)
+Received: from localhost.localdomain (astound-69-42-19-227.ca.astound.net.
+ [69.42.19.227]) by smtp.gmail.com with ESMTPSA id
+ s15sm188254pfd.164.2020.03.25.16.27.48
+ for <openwrt-devel@lists.openwrt.org>
+ (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
+ Wed, 25 Mar 2020 16:27:48 -0700 (PDT)
+From: Rosen Penev <rosenp@gmail.com>
+To: openwrt-devel@lists.openwrt.org
+Date: Wed, 25 Mar 2020 16:27:47 -0700
+Message-Id: <20200325232747.8675-1-rosenp@gmail.com>
+X-Mailer: git-send-email 2.25.1
+MIME-Version: 1.0
+X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
+X-CRM114-CacheID: sfid-20200325_162750_959654_B4A50F7B 
+X-CRM114-Status: UNSURE (   7.31  )
+X-CRM114-Notice: Please train this message.
+X-Spam-Score: -0.2 (/)
+X-Spam-Report: SpamAssassin version 3.4.4 on bombadil.infradead.org summary:
+ Content analysis details:   (-0.2 points)
+ pts rule name              description
+ ---- ----------------------
+ --------------------------------------------------
+ -0.0 RCVD_IN_DNSWL_NONE RBL: Sender listed at https://www.dnswl.org/,
+ no trust [2607:f8b0:4864:20:0:0:0:1043 listed in]
+ [list.dnswl.org]
+ -0.0 SPF_PASS               SPF: sender matches SPF record
+ 0.0 FREEMAIL_FROM Sender email is commonly abused enduser mail
+ provider [rosenp[at]gmail.com]
+ 0.0 SPF_HELO_NONE          SPF: HELO does not publish an SPF Record
+ -0.1 DKIM_VALID_EF Message has a valid DKIM or DK signature from
+ envelope-from domain
+ -0.1 DKIM_VALID_AU Message has a valid DKIM or DK signature from
+ author's domain
+ 0.1 DKIM_SIGNED            Message has a DKIM or DK signature,
+ not necessarily valid
+ -0.1 DKIM_VALID Message has at least one valid DKIM or DK signature
+Subject: [OpenWrt-Devel] [PATCH] fstoools: add define for GLOB_ONLYDIR
+X-BeenThere: openwrt-devel@lists.openwrt.org
+X-Mailman-Version: 2.1.29
+Precedence: list
+List-Id: <openwrt-devel.lists.openwrt.org>
+List-Unsubscribe: <http://lists.infradead.org/mailman/options/openwrt-devel>, 
+ <mailto:openwrt-devel-request@lists.openwrt.org?subject=unsubscribe>
+List-Archive: <http://lists.infradead.org/pipermail/openwrt-devel/>
+List-Post: <mailto:openwrt-devel@lists.openwrt.org>
+List-Help: <mailto:openwrt-devel-request@lists.openwrt.org?subject=help>
+List-Subscribe: <http://lists.infradead.org/mailman/listinfo/openwrt-devel>, 
+ <mailto:openwrt-devel-request@lists.openwrt.org?subject=subscribe>
+Sender: "openwrt-devel" <openwrt-devel-bounces@lists.openwrt.org>
+Errors-To: openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org
+
+This was originally a patch for musl. It is not present in musl as it is a
+GNU extension.
```

Comment:
Please extend the description.

GLOB_ONLYDIR is not supported by musl, it is used to restrict the output of glob to directories only, the code has some extra handling to filter out other files for libc implementations that do not support GLOB_ONLYDIR

Reply:
Replaced with openembedded patch.